core-bundle/src/Security/Authentication/Token/TokenChecker.php

```diff
@@ -56,6 +57,24 @@ public function hasBackendUser(): bool
         return null !== $token && $token->getUser() instanceof BackendUser;
     }
 
+    /**
+     * Checks if a user has a two-factor token (in 2FA progess).
+     */
+    public function hasTwoFactorToken(): bool
+    {
+        $token = null;
+
+        if ($this->hasFrontendUser()) {
+            $token = $this->getToken(FrontendUser::SECURITY_SESSION_KEY);
```

Comment:
@aschempp What about these constants now? Keep or replace?

Reply:
https://github.com/contao/contao/pull/533

Reply:
Fixed in rebase.